stock_delivery_result_import/wizards/stock_delivery_result_import.py

```diff
@@ -72,14 +74,20 @@ def import_stock_delivery_result(self):
                 )
         if not import_log.error_ids:
             for picking, vals in pick_dict.items():
-                carrier_info = vals["carrier_info"]
+                carrier_info = (
+                    picking.carrier_info_id
+                    if picking.carrier_info_id
+                    else vals["carrier_info"]
+                )
                 tracking_refs = list(set(vals["tracking_refs"]))
+                if picking.carrier_tracking_ref:
+                    tracking_refs.append(picking.carrier_tracking_ref)
```

Comment:
This manipulation should be done here https://github.com/qrtl/mi7-custom/pull/87/files#diff-037e9962dee26567028229c7b0c134ac871bc72397305b6865a6ad61f2d2413eL60 instead.